.github/workflows/release.yaml

```diff
@@ -288,6 +289,67 @@ jobs:
           ocm_repos="$(cat prepare-values.d/ocm-repositories)"
           rm -rf prepare-values.d prepare-workflow-values.tar
           echo "ocm-repositories=${ocm_repos}" >> ${GITHUB_OUTPUT}
+      - name: prepare-next-version-operation
+        id: prep-next-version
+        shell: python
+        run: |
+          import os
+
+          import yaml
+
+          import ocm.branch_info
+
+
+          next_version = '${{ inputs.next-version }}'
+
+          if next_version != 'automatic':
```

Comment:
how about putting this check into an `if` (such that GHA will not have to go through the lengths of starting a python-interpreter (admittedly, that comes at the price of adding a condition for reading next-version) :-))

Reply:
agreed